src/js/common/Utilities.js

```diff
@@ -102,21 +94,56 @@ define(["jquery", "underscore"], function ($, _) {
         return [];
       }
 
-      var header_line = text.slice(0, index);
-      var names = header_line.split(",");
+      const headerLine = text.slice(0, index);
+      let names = headerLine.split(",");
 
       // Remove surrounding parens and double-quotes
-      names = names.map(function (name) {
-        return name.replaceAll(/^["']|["']$/gm, "");
-      });
+      names = names.map((name) => name.replaceAll(/^["']|["']$/gm, ""));
 
       // Filter out zero-length values (headers like a,b,c,,,,,)
-      names = names.filter(function (name) {
-        return name.length > 0;
-      });
+      names = names.filter((name) => name.length > 0);
 
       return names;
     },
+
+    // Format the number into a string with better readability.
+    formatNumber(value, min, max) {
```

Comment:
I wrote an expectation that failed:
```
expect(EntityUtils.formatNumber(0.99999, 0, 0.0001)).to.equal("0.0001");
```
I would expect min and max to define the range, but that doesn't appear to be the case, what does min and max represent?

Reply:
Ah yea, the min and max here is misleading. In practice, value should always be in between min and max. And the method only cares about (max - min) to decide how many decimal places to use. I updated the param to just use a `range` instead, and added more details to the JSDocs.